frontend/src/app/components/op-context-menu/handlers/op-columns-context-menu.directive.ts

```diff
@@ -122,6 +122,7 @@ export class OpColumnsContextMenu extends OpContextMenuTrigger {
             this.confirmDialog.confirm({
               text: this.text.confirmDelete,
             }).then(() => {
+              this.wpTableSortBy.clearAllSortings();
```

Comment:
This will modify the state twice, which we should probably avoid

Reply:
👍 I moved it all to one method, where we can directly overwrite the sorting array with the one value.